packages/core/src/simulate.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  AnyActorLogic,
+  AnyStateMachine,
+  EventFromLogic,
+  InternalStateFrom,
+  SnapshotFrom
+} from '.';
+
+export function simulate<T extends AnyActorLogic>(
+  actorLogic: T,
+  options: { input?: any } = {}
+) {
+  const dummyActorContext: any = {
+    self: {}
+  };
```

Comment:
This definitely needs more to be put on it - we need to satisfy the real `ActorContext` type to avoid mistakes further down the road. While working on the other PR I had something like this for it:
```ts
function createMockActorContext(): AnyActorContext {
  const nullActor = createEmptyActor();
  return {
    self: nullActor,
    logger: console.log,
    id: 'root_test',
    sessionId: Math.random().toString(32).slice(2),
    defer: () => {},
    system: nullActor,
    stopChild: () => {}
  };
}
```